src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -15,6 +13,52 @@
 
 use Sonata\ClassificationBundle\Model\CategoryInterface;
 use Sonata\EasyExtendsBundle\Mapper\DoctrineCollector;
+use Sonata\MediaBundle\CDN\CDNInterface;
```

Comment:
are you sure all these Classes are needed?

Reply:
I have updated the file